index.html

```diff
@@ -40,10 +40,12 @@ <h1 class="title">Bouncy Ball</h1>
     <label for="smil" class="label label-smil">SMIL</label>
     <input type="radio" id="mojs" name="approaches" />
     <label for="mojs" class="label label-mojs">mo.js</label>
+    <input type="radio" id="anime" name="approaches" />
+    <label for="anime" class="label label-anime">anime.js</label>
   </form>
   <div class="panes">
     <div class="docs-toggle">
-      <button class="docs-toggle-link button-link" id="toggle">about <span class="demo-name">Vanilla JS</span></button>
+      <a class="docs-toggle-link" id="toggle" href="#">about <span class="demo-name">Vanilla JS</span></a>
```

Comment:
We want to keep this as a button. This was a recent change, and I'm guessing it got mixed up in a rebase.